BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/Implementation/Tracing/SelfDiagnostics/SelfDiagnosticsConfigRefresherTest.cs

```diff
@@ -74,6 +75,50 @@ public void SelfDiagnosticsConfigRefresher_CaptureAsConfigured()
             }
         }
 
+        [TestMethod]
+        public void SelfDiagnosticsConfigRefresher_ReadFromEnviornmentVars_OnlyLogDirectoryWasSet()
+        {
+            Environment.SetEnvironmentVariable("LogDirectory", ".");
```

Comment:
Do you need to unset the variable(s) once each test case is finished? Will it affect other test cases?

Reply:
should run these tests in a try/catch/finally and set the vars to null

Reply:
I've added try/finally to reset the environment variables for the tests; thank you